README.md

```diff
@@ -53,7 +56,7 @@ var foods = [ ["apple","banana","strawberry"], ["pizza","fries","hamburger"] ];
 How would I go about accessing the string "pizza" in the above array?
 
 ```js
-// write code here
+//Unclear on how to access nested arrays...
```

Comment:
I recommend reviewing the solutions branch.  Feel free to ask questions here or in Slack.